pages/login.vue

```diff
@@ -236,20 +236,11 @@
               <a href="http://www.publiq.be" class="footer-link">publiq vzw</a>
             </p>
 
-            <img
-              v-show="$i18n.locale === 'nl'"
-              src="~/assets/vlaanderen-logo.png"
-              class="logo-flanders-full"
-            />
-            <img
-              v-show="$i18n.locale === 'fr'"
-              src="~/assets/flanders-logo.png"
-              class="logo-flanders-full"
-            />
+            <img v-show="isNL" src="~/assets/vlaanderen-logo.png" class="logo-flanders-full" />
+            <img v-show="isFR" src="~/assets/flanders-logo.png" class="logo-flanders-full" />
```

Comment:
Would it make sense to also add a key for the image filename to the translation files, and do something like this?
```
<img src="~/assets/{{ $i18n.t('flanders-logo-image') }}" class="logo-flanders-full" />
```
Or is that not possible/frowned upon?

Reply:
We probably also need a translatable `alt` for this in any case 🤔